src/jestUtils/jestUtils.ts

```diff
@@ -467,9 +470,13 @@ export function fireGestureHandler<THandler extends AllGestures | AllHandlers>(
   componentOrGesture: ReactTestInstance | GestureType,
   eventList: Partial<GestureHandlerTestEvent<ExtractConfig<THandler>>>[] = []
 ): void {
-  const { emitEvent, handlerType, handlerTag } =
+  const { emitEvent, handlerType, handlerTag, enabled } =
     getHandlerData(componentOrGesture);
 
+  if (enabled === false) {
```

Comment:
Do you need this strict type check? Would `if (!enabled) {` work for you then?

Reply:
By default none of the props are initialized and default values are assumed. The default for `enabled` is `true` but the prop itself may be `undefined`, in which case `if (!enabled) {` wouldn't work.

Reply:
It is tricky way to consider boolean with 3 values true/false/undefined. But I got you it will work. Thank you for explanation!